docs/UserGuide.md

```diff
@@ -167,19 +167,19 @@ Example: riskprofile 1 res/a,e,b,d,c,a,d,e
 Adds a new entry to their address book for financial advisors, including personal details such as name, address, occupation, phone number and email.
 
 #### Format:
-* `add n/NAME p/PHONE_NUMBER e/EMAIL o/OCCUPATION a/ADDRESS [t/TAG] [appt/APPOINTMENT_DATE]`
+* `add n/NAME p/PHONE_NUMBER e/EMAIL o/OCCUPATION a/ADDRESS [t/TAG]… [appt/APPOINTMENT_DATE]`
```

Comment:
I think we need to standardize parameter `APPOINTMENT_DATE`.
I notice some part of the UG use `APPOINTMENTDATE` while other parts use `APPOINTMENT_DATE` or `APPOINTMENT DATE`

Reply:
alright, in the meantime, I approve the PR first. I try to fix it in my PR instead!

Reply:
I think when talking about the parameter, definitely not APPOINTMENT DATE. Maybe APPOINTMENT_DATE? I'll check with the group.